### ISSUE TITLE
wallhack chams not working in certain conditions

### ISSUE THREAD

**Issue by a6d6061ed1** [ImpactDevelopment/ImpactClient#1316]
**Describe the bug**
wallhack chams stop working when I get close to the walls

**To Reproduce**
1. enable chams
2. switch mode to wallhack
3. tick all other options in chams section
4. get close to the wall and look for some mobs/players/passives

**Expected behavior**
chams should render entities through the walls

**Screenshots**
N/A

<details> 
  <summary><strong>Additional Client Info</strong></summary>
 

- Impact Version: latest for minecraft 1.12.2
- Using Optifine? yes
- Using LiteLoader? no
- Enabled mods: [click](https://i.imgur.com/f0i8r4j.png)
- Operating System: Win10 Pro x64 1903

</details>

**Additional context**
[video](https://streamable.com/7888a)

**Comment by Cerbiac** [comment]
OptiFine could be the cause.

**Comment by Cerbiac** [comment]
OptiFine does not affect it.
1. Enable Chams [Wallhack]
2. Enable ESP (Optional, helps visualize mobs disappearing from the view of wallhack.)
3. Increase FOV (Optional, rendering stops at the sides of the screens.)
4. Look into a wall in a natural world and notice mobs disappearing and reappearing.

**Comment by usereewrjrjeejr** [comment]
Isn't this the same as #1247?

**Comment by a6d6061ed1** [comment]
> OptiFine does not affect it.
> 1.12.2 Minecraft w/ Impact 4.6 w/ and w/o OptiFine E3.
> 
> 1. Enable `Chams [Wallhack]`
> 2. Enable `ESP` (Optional, helps visualize mobs disappearing from the view of wallhack.)
> 3. Increase FOV (Optional, rendering stops at the sides of the screens for some mobs.)
> 4. Look into a wall and notice some mobs disappearing and reappearing based on distance or distance from edge of screen.

yes it doesn't happen exclusively when player is too close to the wall. it also happens in different situations which I can't really describe precisely

**Comment by usereewrjrjeejr** [comment]
#1241?

**Comment by a6d6061ed1** [comment]
> #1241?

first you linked waypoint suggestion and now some server specific crash? both completely not related

**Comment by usereewrjrjeejr** [comment]
My bad
